src/Shouldly.Tests/ConventionTests/ApprovePublicApi.ShouldlyApi.approved.cs

```diff
@@ -7,156 +7,175 @@ public enum Case
     }
     [System.ObsoleteAttribute("This class is only kept here for backwards compatibility. Please use ShouldAssert" +
         "Exception instead.")]
+    [System.Runtime.CompilerServices.NullableAttribute(0)]
     public class ChuckedAWobbly : System.Exception
```

Comment:
@JosephWoodward This diff is going to look miserable until https://github.com/ApiApprover/ApiApprover/issues/54 is fixed. I don't know if that means this PR should wait?

Reply:
@jnm2 Shouldly is currently in a beta release of a new major version change so if we were going to introduce any breaking changes to the public API then now is a suitable time to do it (providing it didn't break backwards compatibility of course).

Reply:
Cool. I don't think any of these API changes are breaking changes. (Unless I unintentionally changed something besides NRT annotations.) If a project already has `<Nullable>enable</Nullable>` and upgrades to the annotated Shouldly, new warnings might show up now that the compiler knows for sure which inputs and outputs from Shouldly can and can't be null. I believe more of .NET Core is receiving annotations in 3.1, not a major version change.